presentation-api/controlling-ua/PresentationRequest_onconnectionavailable-manual.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>PresentationRequest.onconnectionavailable (manual test)</title>
+<link rel="author" title="Tomoyuki Shimizu" href="https://github.com/tomoyukilabs">
+<link rel="help" href="https://w3c.github.io/presentation-api/#starting-a-presentation">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<p>Click the button below and select the available presentation display, to start the manual test.</p>
+<button id="presentBtn">Start Presentation Test</button>
+
+
+<script>
+    // disable the timeout function for the tests
+    setup({explicit_timeout: true});
+
+    // ----------
+    // DOM Object
+    // ----------
+    var presentBtn = document.getElementById("presentBtn");
+
+    // ------------
+    // Request init
+    // ------------
+    var validUnixDate = new Date().getTime() + String(Math.floor(Math.random() * 1e5)),
+            validPresURL = '../receiving-ua/idlharness.html#__castAppId__=2334D33A/__castClientId__=' + validUnixDate,
+            request = new PresentationRequest(validPresURL);
+
+    // --------------------------------------------------------------------------
+    // Start New PresentationRequest.onconnectionavailable Test (success) - begin
+    // --------------------------------------------------------------------------
+    var startPresentation = function () {
+        promise_test(function (t) {
+            // Note: During starting a presentation, the connectionavailable event is fired (step 20)
+            //       after the promise P is resolved (step 19).
+            return new Promise(function(resolve, reject) {
+                request.onconnectionavailable = function (evt) {
+                    resolve(evt.connection);
+                };
+                request.start().then(function() {
+                    // This test fails if request.onconnectionavailable is not invoked although the presentation is started successfully
+                    t.step_timeout(function() { reject('The connectionavailable was not fired.') }, 5000);
+                }, function() {
```

Comment:
I may be nitpicking here, but wouldn't it be better to replace `function() { reject(...); }` by "reject"?

```js
request.start().then(function() { ... }, reject);
```

This would expose the actual reason for failure in the test result in particular (`NotAllowedError`, `NotFoundError`, etc.) which would help with debugging.